LR_Tourist/Tourist/PresentationHotel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using BLL.Model;
+using BLL.Services;
+using System.Collections.Generic;
+
+namespace TouristConsole
+{
+    public class PresentationHotel
```

Comment:
Why do we need all Presentation services (except PresentationMenu)?
Can you use the services provided by BL?

Reply:
fixed